README.md

```diff
@@ -611,7 +611,7 @@ storage_config = {
 }
 ```
 
-Redis >= 2.6.0 is required as this storage requires [PEXPIRE](https://redis.io/commands/pexpire).
+Redis >= 2.6.12 is required as this storage requires [SET EX](https://redis.io/commands/set).
```

Comment:
I wonder who uses such antique versions anyway (current is 7.0, 2.6 is not listed in [release notes](https://docs.redis.com/latest/rs/release-notes/legacy-release-notes/) but seems to be from 2015).

Or are there two different versioning schemes?

Reply:
Haha, https://gist.github.com/lqez/3944436 11 years ago. Maybe it's just 🦖 old.